### PR TITLE
Added definitions for stripe.js/v3 elements

### PR DESCRIPTION
**Pull request by amritk**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://stripe.com/docs/stripe.js?>>
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dslint/dt.json" }`.

For some reason it kept giving me the `Cannot find name 'HTMLElement'` error, so I had to set one type to any.

### CODE REVIEW

**Comment by amritk** [comment]
whats the meaning of can't parse definition header... ?

**Comment by aozgaa** [comment]
👍 or 👎 , 
@a904guy, @ejsmith, @amritk, @adamcmiel, @jleider?

@amritk , the bot should automatically ping folks to review as I have manually done.

**Comment by jleider** [comment]
How does this change affect v2 consumers? Please see this: https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version regarding major version changes.

**Comment by amritk** [comment]
@jleider I didn't see that, but I wrote it in a way that it won't affect anyone using v2 since the new types are scoped under the namespace "stripe". Should i switch to that method?

**Comment by jleider** [comment]
@amritk My main concern is that we dont break v2 consumers' types. It seems like the canonical way of doing that is to branch off the existing code by moving v2 types to their own directory that way they aren't inadvertently broken by any v3 specific updates in the future.

**Comment by mhegazy** [comment]
@amritk please follow the instructions in https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version for adding a new major version.

the version in the tag is used to publish the @types pakage

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,1666 @@
+{
```
Comment (mhegazy) [inline package-lock.json]: please remove.

File: types/stripe/index.d.ts [hunk 2]
```diff
@@ -1,170 +1,123 @@
...
-}
-
-interface StripeBankTokenParams {
-    country: string;
-    currency: string;
-    account_number: number | string;
-    routing_number?: number | string;
-    account_holder_name: string;
-    account_holder_type: string;
+/// <reference types="stripe/v2" />
```
Comment (mhegazy) [inline types/stripe/index.d.ts]: you should not reference the same package. each one of these represent a version of `@types/stripe`. each should be self contained.

Comment (mhegazy) [inline types/stripe/index.d.ts]: in general, after you copy the exiting to v2, which you have done, you want to add, and not remove.

File: types/stripe/index.d.ts [hunk 3]
```diff
@@ -1,170 +1,123 @@
-// Type definitions for stripe 2.x
+// Type definitions for stripe 3.0
 // Project: https://stripe.com/
-// Definitions by: Andy Hawkins <https://github.com/a904guy/,http://a904guy.com>
```
Comment (mhegazy) [inline types/stripe/index.d.ts]: i would keep the original authors. just add yourself.

**Comment by yuit** [comment]
Thanks for your contribution. This PR has unaddressed comments and cannot be merged in at the time being. For housekeeping purposes we are closing stale PRs. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

**Comment by amritk** [comment]
Alright I will open another pull request for it

**Pull request closed without merging**